ksqldb-engine/src/main/java/io/confluent/ksql/physical/scalablepush/consumer/LatestConsumer.java

```diff
@@ -154,15 +142,20 @@ private void maybeSeekToEnd() {
         consumer.offsetsForTimes(timestamps);
     final Map<TopicPartition, OffsetAndMetadata> offsetAndMetadataMap =
         consumer.committed(topicPartitions);
+    LOG.info("ALAN: offsetAndTimestampMap {}, offsetAndMetadataMap {}", offsetAndTimestampMap,
+        offsetAndMetadataMap);
     for (Map.Entry<TopicPartition, OffsetAndTimestamp> entry : offsetAndTimestampMap.entrySet()) {
       final OffsetAndMetadata metadata = offsetAndMetadataMap.get(entry.getKey());
+      LOG.info("ALAN: metadata {}, entry.getValue() {}", metadata, entry.getValue());
       if (metadata != null && entry.getValue() != null
           && entry.getValue().offset() > metadata.offset()) {
+        LOG.info("ALAN: About to seek seeking to end {}", currentPositions);
         consumer.seekToEnd(topicPartitions);
-        resetCurrentPosition();
+        updateCurrentPositions();
         LOG.info("LatestConsumer seeking to end {}", currentPositions);
         return;
       }
     }
+    LOG.info("ALAN: DID NOT SEEK TO END");
```

Comment:
`I AM GROOT` (a few times 😆 )

in seriousness, if these are helpful logs should we include them at a trace or debug level?

Reply:
Eek.  Added one of them since I think it's helpful.